components/collapse/style/index.less

```diff
@@ -3,8 +3,7 @@
 
 @collapse-prefix-cls: ~"@{ant-prefix}-collapse";
 
-@collapse-header-bg: @background-color-light;
-@collapse-active-bg: @background-color-base;
```

Comment:
same as @collapse-active-bg

Reply:
@collapse-active-bg isn't used anywhere. Unless there's some kind of logic auto-attaching it to something, it looks like @collapse-active-bg is just cruft. I can restore it if you'd like, but it's not doing anything and it would be cleaner to remove it than leave it around looking like it's doing something.

Reply:
Unless it's *supposed* to be attached to something, but I don't have insight into that intent or why it's no longer attached.

Reply:
It is Ok in that case.